src/session/content/chat_action_bar.rs

```diff
@@ -560,3 +566,16 @@ fn message_bar_visibility_in_groups(
     )
     .upcast()
 }
+
+async fn parse_markdown(
```

Comment:
I think @melix99 has already mentioned it. The markdown parsing now only works in the `ChatActionBar` but not in the `SendPhotoDialog`. So the parsing should happen directly in the `MessageEntry` to make it usable.

Reply:
Ok, I'll do it tomorow, For this message entry will need `WeakRef<Session>` to get `client_id()`

Reply:
Well, I've been thinking about this problem and I think passing `Session` or `client_id` to the contructor would conflict with `Default` implementation
```rust 

impl Default for MessageEntry {
    fn default() -> Self {
        Self::new()
    }
}
```
I'm not sure if it's worth it to modify the constructor if there is a simpler workaround and the text is being parsed only in two places (ChatActionBar, SendPhotoDialog) which has already access to `client_id()`.

Reply:
`ChatActionBar` which contains the `MessageEntry` also contains the `Chat`. `Chat` contains the `Session`. `Session` contains the `client_id`.

`SendPhoto` also contains both the `Chat` and the `MessageIcon`

So you also need to add a glib GObject property in `chat` to `MessageEntry` and bind that to the `chat` property of the parent widget.

https://github.com/melix99/telegrand/blob/f9a5bd88162211846d392f8db72a3bd7aff90220/data/resources/ui/content-chat-action-bar.ui#L21

https://github.com/melix99/telegrand/blob/f9a5bd88162211846d392f8db72a3bd7aff90220/data/resources/ui/content-send-photo-dialog.ui#L33

If you have implemented the property, you can just bind it.
```xml
                <child>
                  <object class="MessageEntry" id="caption_entry">
                    <property name="hexpand">True</property>
                    <property name="placeholder-text" translatable="yes">Caption</property>
                    <binding name="chat">
                      <lookup name="chat">ContentSendPhotoDialog</lookup>
                    </binding>
                  </object>
                </child>
```